### PR TITLE
new: introduce allowedTypes

### PR DESCRIPTION
**Pull request by loresuso**
Signed-off-by: Lorenzo Susini <susinilorenzo1@gmail.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

> /kind flaky-test

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area library

/area cli

> /area tests

> /area examples

**What this PR does / why we need it**:
Falcoctl is able to download any kind of artifacts, including shared libraries that could be potentially used by Falco and loaded. For security reasons, we want to let users be able to decide which kind of artifacts falcoctl is allowed to pull by means of a whitelist. If the list is empty everything is allowed for ease of use. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: cmd/artifact/install/install.go [hunk 1]
```diff
@@ -128,6 +144,8 @@ func NewArtifactInstallCmd(ctx context.Context, opt *options.CommonOptions) *cob
 		"directory where to install rules. Defaults to /etc/falco")
 	cmd.Flags().StringVarP(&o.pluginsDir, "plugins-dir", "", config.PluginsDir,
 		"directory where to install plugins. Defaults to /usr/share/falco/plugins")
+	cmd.Flags().Var(&o.allowedTypes, "allowed-types",
+		"whitelist of artifacts type that can be installed. If an empty list is passed (or configured), everything is allowed")
```
Comment (LucaGuerra) [inline cmd/artifact/install/install.go:148]:
```suggestion
		"comma-separated list of artifact types that can be installed. If not specified, all types are allowed. Example: \"rulesfile,plugin\"")
```

Comment (LucaGuerra) [reply cmd/artifact/install/install.go:148]: I think this phrasing is a little clearer but the flag name and everything as implemented makes sense to me :+1:

Comment (loresuso) [reply cmd/artifact/install/install.go:148]: Agreed but it isn't only a comma-separated list, you can also specify the flag multiple times. Any other hint?

Comment (LucaGuerra) [reply cmd/artifact/install/install.go:148]: Thanks @loresuso , I think your solution is the best since it explains all ways to use the flag!

Comment (loresuso) [reply cmd/artifact/install/install.go]: 👍 updated!

**Review by leogr — APPROVED** [review]
:+1:

**Pull request merged**